### DISCUSSION TITLE
Popup message now working properly in WINUI 3 with UNO

### DISCUSSION THREAD

**Discussion by amit7775** [unoplatform/uno#8847] [General]
Below error coming when try to open popup ( NoDataPopup.IsOpen = true;)
Catastrophic failure (0x8000FFFF (E_UNEXPECTED))

but same code working fine on button click event
   private void btnclose_Click(object sender, RoutedEventArgs e)
        {
            if (!ClosePopup.IsOpen)
            { ClosePopup.IsOpen = true; }

        }

and when we write same piece of code on if else condition then its giving error
If(result==true)
{
}
else
              { 
                       if (!NoDataPopup.IsOpen)
                    { 
                        NoDataPopup.IsOpen = true; **// Above Error coming in this line**
                    }

                }

**Comment by jeromelaban** [comment]
Thanks. The error you're getting is from WinAppSDK, not from Uno. It's likely that you can take a look at https://github.com/microsoft/microsoft-ui-xaml/issues and find your solution.
